src/Value/Line.php

```diff
@@ -40,6 +46,24 @@ public function raw(): string
         return $this->raw;
     }
 
+    public function rawU(): UnicodeString
```

Comment:
What about adding a parameter to `raw()` method `raw($asUnicodeString = false)` ?

Same for `clean()` 🤔 

Or rename `cleanU()` -> `cleanAsUnicodeString()` ? Imo the `U`-suffix is not very clear 🤔

Reply:
Then I would prefer using `*asUnicodeString()`

Reply:
`clean` and `raw` now returns a `UnicodeString`-object